packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,440 @@
+* `subject_token_type`: Identifies the type of token in the `subject_token` parameter. Value: `urn:ietf:params:oauth:token-type:id_token`
+* `scope`: The scopes that you want to be included in the access token.
+* `audience`: Identifies the audience of the authorization server. Locate the value for the `audience` parameter in the Admin Console. From the left navigation pane, select **Security** > **API**. On the **Authorization Server** tab, select the pencil icon for the authorization server that you need the `audience` value for.
+
+If the request is successful, the response returned includes the following tokens: `id_token`, `refresh_token`, and `access token`.
+
+> **Note:** You can pass an expired ID token as part of the token exchange grant as long as the `device_secret` (`sid`) that the `id_token` is associated with is still valid.
+
+**Example response**
+
+```JSON
+{
+    "token_type": "Bearer",
+    "expires_in": 3600,
+    "access_token": "eyJraWQiOiJZQlRCZUNBOThRa3pSUGdBQzZSS3BaM0J5ZlRwTXRMMHRITkp1LTQ4VVVFIiwiYWxnIjoiUlMyNTYifQ.eyJ2ZXIiOjEsImp0aSI6IkFULnhtbDhfRlBDTFhlOHpoX2ZFLWhkZWZPcm5jbHAzVng4WDRrRVpJTXcxZk0iLCJpc3MiOiJodHRwczovL2NocmlzdGluZWNob3ktb3AyLm9rdGFwcmV2aWV3LmNvbS9vYXV0aDIvZGVmYXVsdCIsImF1ZCI6ImFwaTovL2RlZmF1bHQiLCJpYXQiOjE2MjgwMzI5MzAsImV4cCI6MTYyODAzNjUzMCwiY2lkIjoiMG9hMWpncWxxbXJiT01zR3AweDciLCJ1aWQiOiIwMHV5MjcyZ2RkTFVER2pGYTB4NiIsInNjcCI6WyJvcGVuaWQiLCJkZXZpY2Vfc3NvIl0sInN1YiI6ImNocmlzdGluZS5jaG95QG9rdGEuY29tIn0.LnenHhDWnaDDxf-0_NjXV93OAuG9pxAcEnee00KLeR1LB1X3mPwtN1C-yykMd3sbkUfOhHLEBD1lPfN2nNsYffu4peO-CFXu4viuihdZbyfrKw1-a6OM4AU8jw5HV31dElDz1yT2HVPv5vV0j3W9JKBm-A-PCuYZmzaEDsw3uMtFfIH6QOghRzOUjMzhaQ3QKERyles0nTfOOp_5KempYdwnDPq-_7kRCUykYDHg_YGJ_FCvTEVc6E5blRy2EJ28IViPbHLw0-dqt1S8OEU6Ez3dLGgPHmbEqX1ZxTX_1bRr76kZmgjMnfm4eEwRd3FIRnkm_xh8AkLnmQ2F6hEVUw",
+    "scope": "openid device_sso",
+    "id_token": "eyJraWQiOiJZQlRCZUNBOThRa3pSUGdBQzZSS3BaM0J5ZlRwTXRMMHRITkp1LTQ4VVVFIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiIwMHV5MjcyZ2RkTFVER2pGYTB4NiIsInZlciI6MSwiaXNzIjoiaHR0cHM6Ly9jaHJpc3RpbmVjaG95LW9wMi5va3RhcHJldmlldy5jb20vb2F1dGgyL2RlZmF1bHQiLCJhdWQiOiIwb2ExamdxbHFtcmJPTXNHcDB4NyIsImlhdCI6MTYyODAzMjkzMCwiZXhwIjoxNjI4MDM2NTMwLCJqdGkiOiJJRC55VDFjVFZVdkV0STZmenBETzRYZkVadk5xdjVlVUhwT0IwUC1mMmVPcERBIiwiYW1yIjpbInB3ZCJdLCJpZHAiOiIwMG95MjcyY3NhR0NNamRPVjB4NiIsInNpZCI6IjEwMm9SZ0VXeDVsUlRXSGlsRW9HZmVkNHciLCJhdXRoX3RpbWUiOjE2MjgwMjcxMzcsImF0X2hhc2giOiJGVy1ZclFoXzhQOThmQ1NqUEQxQW1BIiwiZHNfaGFzaCI6IjN0Z3AtUFVuX213MUtkZHdGay1TYUEifQ.lHclmmSNV1ns2GGJCxEBRZpMCp_xAePqAZQvnwJ5207Gr5ohbmPkXYmgI6zA9STTyfPgiMVotJG0Iir2Ir9Ke9PBvcd6fvUTZPZTzg3McMBfRoaF2ORYO-1qoUIIEGUDtYqiUosmc_OR27U-AS6TZfnMo4Ed7KIAIHlPgzJDlAFCCa1vx-ImUCt0czP8eehEqSDE1nB1oPj3uvGpjsJxj4wwFvGXqGemHpWDbvTDvwNv5dGm05q6LNYZjJACyegtIagvKgFv5mZquR-tlXIS6SEynFNsNP_YEf-gNZ36I1y-wk6tjmTgYijcZPcvVl9PvE4giokjXVjPsjP3WGQWgA",
+    "device_secret": "kyLmBvfjMPnMGHtBhW9nKz0tLgJc5xbLay1ujkvOQIk="
```

Comment:
This does not belong here; we only return the `device_secret` from the initial /token call with the authorization code grant, not on token exchange calls to the /token endpoint.